package.json

```diff
@@ -44,6 +44,7 @@
     "url": "https://github.com/opensearch-project/oui.git"
   },
   "resolutions": {
+    "**/semver": "^7.5.3",
```

Comment:
Would like @andreymyssak or @SergeyMyssak opinion on this. I tried explicitly adding prefixes for all deps that have it in their tree but it was a huge number of additions and I couldn't get all the warnings to go away from `yarn audit`. Would this be good or should we do it in a different way?

Reply:
@BSFishy I tried that too, but for some reason not all warnings go away, there is no other way, so let's leave it as it is